.github/workflows/ci.yml

```diff
@@ -39,11 +38,36 @@ jobs:
           --health-interval 10s
           --health-timeout 5s
           --health-retries 5
-
     steps:
     - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
     - uses: ./.github/actions/setup
-    - name: ✨ Verify quality
-      run: make quality
     - name: 🤹 Django tests
       run: make test
+
+  quality:
+      runs-on: ubuntu-latest
+      env:
+        DJANGO_SETTINGS_MODULE: config.settings.test
+        REQUIREMENTS_PATH: requirements/test.txt
+      concurrency:
+        group: ${{ github.workflow }}-quality-${{ github.ref_name }}-${{ github.event.pull_request.number || github.sha }}
+        cancel-in-progress: true
+      services:
+        postgres:
+          # Docker Hub image
+          image: postgis/postgis:17-master
+          env:
+            POSTGRES_PASSWORD: password
+          volumes:
+            - /var/run/postgresql:/var/run/postgresql
+          # Set health checks to wait until postgres has started
+          options: >-
+            --health-cmd pg_isready
+            --health-interval 10s
+            --health-timeout 5s
+            --health-retries 5
```

Comment:
Il y a sûrement moyen de partager ça avec une référence YAML?
Un truc comme ça https://yaml.org/spec/1.2.2/#example-node-for-sammy-sosa-appears-twice-in-this-document

Reply:
GH ne les gèrent pas :(

> The workflow is not valid. .github/workflows/ci.yml: Anchors are not currently supported. Remove the anchor 'concurrency'

Reply:
Dommage :(